app/src/utils/Utils.ts

```diff
@@ -116,3 +116,18 @@ export const getLogOutUrl = (config: IConfig): string | undefined => {
 
   return `${config.SITEMINDER_LOGOUT_URL}?returl=${keycloakLogoutRedirectURL}&retnow=1`;
 };
+
+export const getFormattedFileSize = (fileSize: number) => {
```

Comment:
Could add a couple of tests for this function, should be a quick win

Reply:
Agreed!! 🥇 
is it ok if it is included in the next pr @anissa-agahchen is putting up with unit tests?